File: samples/conference/management-bc/src/main/java/com/microsoft/conference/management/domain/Events/ConferenceCreated.cs
using System;

namespace ConferenceManagement
{
    [Serializable]
    public class ConferenceCreated : ConferenceEvent
    {
        public ConferenceCreated() { }
        public ConferenceCreated(ConferenceInfo info)
            : base(info)
        {
        }
    }
}

File: samples/conference/management-bc/src/main/java/com/microsoft/conference/management/domain/events/ConferenceCreated.java
package com.microsoft.conference.management.domain.events;

import com.microsoft.conference.management.domain.models.ConferenceInfo;

public class ConferenceCreated extends ConferenceEvent {
    public ConferenceCreated() {
    }

    public ConferenceCreated(ConferenceInfo info) {
        super(info);
    }
}
